agcaballero/portfolio/src/main/webapp/script.js

```diff
@@ -177,14 +177,14 @@ console.log(constructSearchLink('Fun Search Term, But Not This Part'));
 
 /** Gets comments from data tag and updates "Comments" page with it */
 async function getComments() {
-  let maxComments = document.getElementById("max-comments").value;
-  if(maxComments == "")
-    maxComments = 50; // set default max comments to 50
-  
+  let maxComments = document.getElementById('max-comments').value;
+  if (maxComments == '') maxComments = 50;  // set default max comments to 50
```

Comment:
Is this needed, since the Java code also sets a default of 50 in the getRequestParameter method? There's a downside of having defaults like this set in multiple places, which is that if you ever want to change the defaults you've got to go hunt down the different places where they're set.

Reply:
That makes a lot of sense. I will change that (it should be in comments.js file after I change it).

Reply:
I decided to put the defaults in the JS file, but remove them from the Java file. This way they're only in one place. Placing them in a Java file seems to cause bugs.

Reply:
Done